apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useAddMember.ts

```diff
@@ -76,7 +76,7 @@ export function useAddMember({ device }: { device: Device | null }) {
             onEndRequestUserInteraction: () => setUserDeviceInteraction(false),
           },
           undefined,
-          trustchainRef?.current?.rootId,
+          trustchainRef?.current ?? undefined,
```

Comment:
`rootId` is always defined on the `Trustchain` type.
I think the `?.` was just a leftover from some previous code (unecessary `?` are not detected by the linter ATM).

Reply:
in this case `trustchainRef?.current` is sufficient, if current is not defined it will returns an undefined. I don't see a need for nullish coalescing operator here(also in useAddMember) . can you test without please ?

Reply:
`trustchainRef?.current` is always defined but it's value might be `null` that's why I put the `?? undefined` (because `trustchain` is an optionnal arg) otherwise I could make it `trustchain?: Trustchain | null` if you prefer

Reply:
hummm ! i understand ok no worries. Thanks

Reply:
Oh actually the `?.` wasn't leftover... I just realised that `null?.prop` is `undefined` and not `null` !
Which kind of makes sense when written like that !

Reply:
My bad the first `?.` is a leftover I'll remove it now ! Sorry I confused myself 🤦